ui/src/style/pdf-viewer.less

```diff
@@ -2,11 +2,14 @@
  * Move view of PDF page inward when the drawer is open.
  * TODO(andrewhead): Find a way to animate drawer even when left sidebar is open.
  */
-#viewerContainer {
+.page {
   transition-property: "padding-right";
 
   &.drawer-open {
-    padding-right: @drawer-paper-clipping-width + @margin * 2;
+    padding-right: @drawer-paper-clipping-width + @margin * 4;
+    // A very unfortunate side effect of using padding is that it 
+    // will distort the border image if the user shrinks the page too much.
+    border-image: none;
```

Comment:
Hahah I don't quite understand why, but the PDF's themselves don't actually have borders with a drop shadow, but just a border-image *of* a drop shadow... it's quite strange. But adding padding will distort border-images as the CSS box model states that the border is outside of the padding

Reply:
It's not really noticeable in my opinion so I just removed it, but we can perhaps look into trying to keep the image? It might be a difficult solution, not sure.

Reply:
@zkirby would any of these fixes work?
* setting the `margin` property instead of the `padding` property?
* move the padding to a child element or parent element?
Those have sometimes fixed issues for me where the trying to set spacing between elements has affected those elements' bounding boxes.

Reply:
1. Margin will unfortunately not work (I've tried). 
2. I've tried moving it to a child and doing so will unfortunately do the same thing, but I might be able to try moving it to a parent?

Reply:
Although I suspect that it will not work which is why I moved it to the page in the first place hahaha. Although it's been long enough I have forgotten hahaha